coalib/bears/requirements/PackageRequirement.py

```diff
@@ -6,6 +7,14 @@ class PackageRequirement:
     """
     This class helps keeping track of bear requirements. It should simply
     be appended to the REQUIREMENTS tuple inside the Bear class.
+
+    Two PackageRequirements should always be equal if they have the same
+    manager, package and version:
+
+    >>> PR1 = ('pip', 'coala_decorators', '0.1.0')
+    >>> PR2 = ('pip', 'coala_decorators', '0.1.0')
+    >>> PR1 == PR2
```

Comment:
I think it's generally a good habbit to make variables lower-case, also inside doctests, though this is just preference^^